core/src/main/java/org/jdbi/v3/core/statement/ParsedParameters.java

```diff
@@ -78,4 +79,29 @@ public String toString() {
             + ", parameterNames=" + parameterNames
             + '}';
     }
+
+    /**
+     * A static factory of named {@link ParsedParameters} instances.
+     * List given in argument must contain parameter names from SQL.
+     * Names from a list must be stripped of a colon which stands
+     * before variable in unprocessed SQL.
+     *
+     * @param names the parameter names from SQL statement
+     * @return New {@link ParsedParameters} instance
+     */
+    public static ParsedParameters named(List<String> names) {
+        return new ParsedParameters(false, names);
+    }
+
+    /**
+     * A static factory of positional {@link ParsedParameters} instances.
+     * The count given in the argument must indicate how many positional
+     * parameters are available in SQL.
+     *
+     * @param count the number of positional parameters in SQL statement
+     * @return New {@link ParsedParameters} instance
+     */
+    public static ParsedParameters positional(int count) {
+        return new ParsedParameters(true, Collections.nCopies(count, "?"));
```

Comment:
Neat, I didn't know about the `nCopies` method

Reply:
Indeed. It's really nice because it does not create collection filled with N elements, but a singleton list where every index from 0..N-1 points the same element and effectively reduce memory consumption.